From 9c5ed3204bdb722b5624f6b9dbfba240a50b336c Mon Sep 17 00:00:00 2001
From: fanquake <fanquake@gmail.com>
Date: Fri, 12 Jan 2024 10:25:27 +0000
Subject: [PATCH] build: set minimum required CMake to 3.5

Avoids:
```bash
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.
```

Upstream-Status: Backport [https://github.com/miniupnp/libnatpmp/pull/43]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3034df8..e7cffe5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(natpmp C)
 
 add_definitions(-DNATPMP_STATICLIB -DENABLE_STRNATPMPERR)
